client/tests/e2e/position.js

```diff
@@ -67,8 +67,11 @@ test.serial("Move someone in and out of a position", async t => {
   const $vacantPositionRows = await $$("#vacantPositions table tbody tr")
   let $positionToFillCell
   for (const $row of $vacantPositionRows) {
-    const [$billetCell, $advisorCell] = await $row.findElements(By.css("td"))
+    const [$billetCell, $posRoleCell, $advisorCell] = await $row.findElements(
+      By.css("td")
+    )
     const billetText = await $billetCell.getText()
+    await $posRoleCell.getText()
```

Comment:
Maybe `const roleText = await $posRoleCell.getText()` and add a check for `roleText` in the `if (billetText === …)` below (pass the expected value for this position as an additional parameter)?

Reply:
OK